src/start/RealmScreen.js

```diff
@@ -74,21 +75,22 @@ class RealmScreen extends React.Component {
     const { progress, realm, error } = this.state;
 
     return (
-      <Screen title="Welcome" keyboardAvoiding>
+      <Screen title="Welcome to Zulip" keyboardAvoiding>
         <ScrollView
           ref={(scrollView) => { this.scrollView = scrollView; }}
           centerContent
           keyboardShouldPersistTaps="always"
           onContentSizeChange={() => this.scrollView.scrollToEnd({ animated: true })}
         >
           <View style={[styles.container, moreStyles.container]}>
-            <Label text="Your server URL" />
+            <Image style={[styles.image]} source={require('../../static/img/bus.png')} />
+            <Label text="Server URL" />
             <Input
               style={styles.field}
               autoFocus
               autoCorrect={false}
               autoCapitalize="none"
-              placeholder="Server URL"
+              placeholder="https://your-server-url.zulipchat.com"
```

Comment:
We have discussed before and agreed that having an actual valid value as placeholder to be undesirable.